### DISCUSSION TITLE
Lighthouse tests on the crypto exchange website Binance.com give varying scores and persistent issues that I am not sure how to interpret.

### DISCUSSION THREAD

**Discussion by dxdt-k** [GoogleChrome/lighthouse#15797] [General]
**Summary**

I have been running Lighthouse on various pages at various occasions and found it gives varying results, which prompted me to look closely. Below I attach a typical Lighthouse results on one of the trading page (https://www.binance.com/en/trade/BNB_USDT?type=isolated). 

There are persistent issues that I am not sure how to interprets. i.e.

" There were issues affecting this run of Lighthouse:

No ads were requested when fetching this page.
No ads were rendered when rendering this page.
The GPT tag was not requested.
No ads were rendered when rendering this page.
There may be stored data affecting loading performance in this location: IndexedDB. Audit this page in an incognito window to prevent those resources from affecting your scores.
The page loaded too slowly to finish within the time limit. Results may be incomplete. "

Therefore, I'm not are how to interpret the rest of the results, i.e I'm not are whether the results are in deed the performance of the page, or they are in fact, the results of me inappropriately performing the test. I guess my my question is do these warning issues responsible for the results I got. There are general security concerns regarding the results themselves when consider the nature of the page. But I'm, not sure if here is the right place to discuss them. Nevertheless, Lighthouse is a great tool to help users use the platform securely, for which I would like to take this opportunity to thank everyone involves. I would like to also congratulate all contributors for building such useful tool.         
 
[www.binance.com_2024-01-31_21-28-55.json](https://github.com/GoogleChrome/lighthouse/files/14117262/www.binance.com_2024-01-31_21-28-55.json)
[Lighthouse Report.pdf](https://github.com/GoogleChrome/lighthouse/files/14117269/Lighthouse.Report.pdf)

**Comment by adamraine** [comment]
> No ads were requested when fetching this page.
> No ads were rendered when rendering this page.
> The GPT tag was not requested.
> No ads were rendered when rendering this page.

All of these errors come from the [publisher ads plugin](https://github.com/googleads/publisher-ads-lighthouse-plugin) and should not affect performance or any other default category. If you have more questions then I would suggest asking in that repo.

> There may be stored data affecting loading performance in this location: IndexedDB. Audit this page in an incognito window to prevent those resources from affecting your scores.

This means that data in IndexedDB was not cleared before starting Lighthouse and could affect the page state/performance. The actual impact is highly dependent on your setup, but this best way to reduce the impact is to test in an incogntio page.

> The page loaded too slowly to finish within the time limit. Results may be incomplete. "

In general, this shouldn't affect performance *metrics* since the report was able to measure those. However, there could be optimizations and improvements that Lighthouse misses because the page didn't finish loading.

> I guess my my question is do these warning issues responsible for the results I got.

Can you rephrase this question I'm not sure what you're asking.
